File: backend/src/main/java/cohappy/backend/model/dto/request/ModifyHouseAdvertisementDTO.java
package cohappy.backend.model.dto.request;

import cohappy.backend.model.HouseState;
import lombok.Data;

import java.util.List;

@Data
public class ModifyHouseAdvertisementDTO {
    private List<byte[]> immages;
    private String houseCode;
    private HouseStateDTO state;
    private String description;
}

File: frontend/app/src/main/java/cohappy/frontend/model/dto/request/ModifyHouseAdvertisementDTO.kt
package cohappy.frontend.model.dto.request

import cohappy.frontend.model.dto.HouseStateEnum
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class ModifyHouseAdvertisementDTO(
    val houseCode: String,
    val state: HouseStateEnum,
    var description: String? = null
)
